example/wescheme-example.js

```diff
@@ -38,6 +38,7 @@ document.getElementById('mode').onchange = function(e) {
     cm.doc.clearHistory();
 
     const options = {
+      search: document.getElementById('search'),
```

Comment:
Can the editor create this node programmatically? This should be as easy as possible to integrate into an existing CM installation.

Reply:
Totally, will do that.

Reply:
Done.